utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -691,6 +713,74 @@ int main(int argc, char **argv)
 
         break;
     }
+    case 82:
+        {
+            test_case_name = "ricap";
+            Rpp32u initialPermuteArray[images];
+            Rpp32u permutedArray1[images] ;
```

Comment:
For the permutedArrays, I would much prefer having a typedef struct defining 4 Rpp32us (for the src1/2/3/4). An array of structures should help. We can discuss on a call.